pom.xml

```diff
@@ -36,6 +37,53 @@
         <vitess.vtctld.grpc.port>15999</vitess.vtctld.grpc.port>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <!-- Debezium artifacts -->
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-connector-vitess</artifactId>
```

Comment:
What is the reason to add `debezium-connector-vitess` dependency in `dependencyManagement`?